paperqa/agents/task.py

```diff
@@ -86,6 +86,8 @@ def __init__(
         )
         self._evaluation_callback = evaluation_callback
         self._rewards = rewards
+        self.answer = ""
```

Comment:
Fwiw this is now in some way duplicated across both the `evaluation_from_answer: Callable` passed in, and these new instance attributes.

We should attempt to fix this duplication when fixing https://github.com/Future-House/paper-qa/issues/761